src/finn/transformation/fpgadataflow/set_folding.py

```diff
@@ -174,6 +175,7 @@ def apply(self, model):
                 if op_type in [
```

Comment:
minor: in L159 this is used instead of listing the variants explicitly:
`if swu_node.op_type.startswith("ConvolutionInputGenerator"):`

suggest to either use that style here, or change L159 to also use the explicit listing to keep both of them consistent.